src/browser/index.html

```diff
@@ -4,6 +4,26 @@
 <head>
   <meta charset="UTF-8">
   <link rel="stylesheet" href="../node_modules/bootstrap/dist/css/bootstrap.min.css">
+  <style>
+
+    /*
+     * Note:
+     *
+     * Unusual placement of these styles is so that they activate while the loading
+     * screen is up rather than applying after React initializes to prevent a flash
+     * of unstyled content.
```

Comment:
Probably I have never noticed FOUC in the desktop app. And I think CSS is loaded [here](https://github.com/mattermost/desktop/blob/3fdcb64358909fa530f0ca9e1a69a9e784cb99f3/src/browser/index.jsx#L3) before rendering React compoentns.
Would you provide some gif or picture to know what you saw?

Reply:
This gif shows how it behaves with all my CSS changes removed from index.html and moved into TabBar.css like this:

```css
.TabBar .teamTabItem-label {
  display: inline-block;
  font-weight: bold;
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
  width: 117px;
}
```

![screencast 2018-04-15 10-28-04](https://user-images.githubusercontent.com/419567/38779583-7309300e-4098-11e8-9494-e039203aa261.gif)

At the start of the gif, the app is closed and I open it fresh.

After it is opened, you can see a succession of different styles applied as different CSS files load.

I poked around for a while and the best spot I could find to place the CSS was right in index.html. That was the only way I could suppress the FOUC.

Reply:
Ah, I get what you're going for now. The fact that original `teamTabItem-label` class wasn't being applied until later (when unread messages were detected) created this apparent FOUC in my test case.

When I applied my new CSS (sans the bold rule) using the new selector I wrote to TabBar.css, the FOUC went away. My latest commit should resolve that.